### ISSUE TITLE
Kolbjorn Ironcleave (Karaz Drengi lord in KV) gives wrong defense quest

### ISSUE THREAD

**Issue by SilverWF** [WarEmu/WarBugs#15575]
Kolbjorn Ironcleave (Karaz Drengi lord in KV) gives wrong defense quest - he asks to defend Kazad Dammaz keep, while this keep is actually enemy's
Of course, kills made while in the Karaz Drengi area, doesn't count for that quest.

**Expected behavior and actual behavior:**
Expected: Keep lord give quests to defend his own keep
Actual: Kolbjorn Ironcleave gives a quest to defend another (enemy) keep.

**Steps to reproduce the problem:**
Just take quest from the lord

**Testing Screenshots/Videos/Evidences (always needed):**
![2020 01 28-20 24](https://user-images.githubusercontent.com/16439840/73366843-a5702b80-42bf-11ea-9a17-15944bbd1ce0.jpg)

**Comment by Qorth** [comment]
Agree, quest offered at Karaz Drengi is to kill at Kazad Dammaz and this is the same for Order and Destruction.
